### PR TITLE
Added support for building with Bazel prerelease version.

### PR DESCRIPTION
**Pull request by haberman**

### CODE REVIEW

**Comment by hlopko** [comment]
git clone git@github.com:bazelbuild/bazel.git && cd bazel && bazel build //src:bazel 

to get the prerelase bazel version :)

File: bazel/upb_proto_library.bzl [hunk 1]
```diff
@@ -99,7 +99,8 @@ def _cc_library_func(ctx, name, hdrs, srcs, dep_ccinfos):
             linking_context = linking_info.linking_context,
         )
 
-    if not versions.is_at_least("0.25.2", bazel_version):
+    # Assume Bazel prerelease versions (version == "") are ok.
+    if not versions.is_at_least("0.25.2", bazel_version) or bazel_version == "":
```
Comment (hlopko) [inline bazel/upb_proto_library.bzl]: Shouldn't it be `if not_fresh_enough and not_dev_build?`

Comment (haberman) [reply bazel/upb_proto_library.bzl]: Ah yes, I coded that too quickly. :) Done.

**Comment by haberman** [comment]
Withdrawn in favor of https://github.com/protocolbuffers/upb/pull/258.

**Pull request closed without merging**